Assets/Scripts/Spells/Abstract/ShootingSpell.cs

```diff
@@ -8,19 +8,16 @@ namespace Assets.Scripts.Spells
 {
     public abstract class ShootingSpell : Spell
     {
-        protected GameObject WizardGameObj;
-
         public override void Activate()
         {
             TurnManager turnManager = GameObject.FindObjectOfType<TurnManager>();
```

Comment:
@czkafa
Starałem się to naprawić usuwając tę linijkę i wprowadzając parametr publiczny TurnManager. Przekazałem prefab TurnManager i wrzuciłem go na scenę ale leciał mi exception na klasach BattleGUI. Jak masz chęci mógłbyś to przerobić i wrzucić poprawki pod tym PRem ? :)

Reply:
Wg mnie cala budowa tego szpela nie jest poprawana powinnien on byc wyrzucany na zewnatrz(przed aktywacja spela powinien on byc tworzony i dawana mu sila) czyli np dajesz wizardowi punkty z ktorego beda wystrzeliwane czary(zawsze zeby bylo z przodu) do spela albo do fabryki speli przychodza tylko argumenty sila i itp